services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/presigner/DefaultS3Presigner.java

```diff
@@ -117,6 +120,11 @@ private DefaultS3Presigner(Builder b) {
         super(b);
 
         this.serviceConfiguration = b.serviceConfiguration != null ? b.serviceConfiguration : DEFAULT_S3_CONFIGURATION;
+        if (serviceConfiguration.checksumValidationEnabled()) {
+            log.warn(() -> "Please note that the checksum validation is enabled by default, so if you are passing a "
+                           + "default S3Configuration, the presigned url won't be browser-executable.");
```

Comment:
Can we change it to something like the following?
```
The provided S3Configuration has ChecksumValidationEnabled set to true. Please note the pre-signed request can't be executed using a web browser if checksum is enabled.
```
Because they could pass a non-default S3Configuration with checksumValidation enabled explicitly

On second thought, maybe we should change the level to `debug` since this doesn't concern customers who do not need it to be browser executable.

Reply:
Sounds reasonable, will do the change.